### PR TITLE
Update jenkins toolbox to latest build with kubectl v1.15.3

### PR DESCRIPTION
**Pull request by kuzaxak**
Redo merging of https://github.com/salemove/pipeline-lib/pull/91

### CODE REVIEW

**Comment by kuzaxak** [comment]
!deploy

File: vars/toolboxContainer.groovy [hunk 1]
```diff
@@ -0,0 +1,8 @@
+def call(Map args) {
```
Comment (deiwin) [inline vars/toolboxContainer.groovy]: You need to default this to an empty map ` = [:]`

Comment (deiwin) [reply vars/toolboxContainer.groovy]: Otherwise it tries to `<< null`

**Comment by kuzaxak** [comment]
!deploy

**Comment by kuzaxak** [comment]
!deploy

**Comment by kuzaxak** [comment]
!deploy

**Comment by kuzaxak** [comment]
!deploy

File: src/com/salemove/Deployer.groovy [hunk 2]
```diff
@@ -491,7 +491,7 @@ class Deployer implements Serializable {
     script.withCredentials([script.string(credentialsId: 'eks-deployer-iam-role', variable: 'role')]) {
       script.inDockerAgent(
         name: 'deployer',
-        containers: [script.interactiveContainer(name: containerName, image: 'salemove/jenkins-toolbox:327930e')],
```
Comment (deiwin) [inline src/com/salemove/Deployer.groovy]: Needs to be `script.toolboxContainer(name: containerName)`

**Comment by kuzaxak** [comment]
!deploy

**Pull request merged**